### ISSUE TITLE
Crash Detection Wrongly Detects Crash!

### ISSUE THREAD

**Issue by mjgerace** [barneygale/mark2#96]
I am using spigot build #1009, and the latest Mark2 build. Mark2 seems to think the server has crashed, when indeed it hasn't. Here is a mark2 log for example: http://pastebin.com/8hPP1TGJ

Please fix this.

Michael

**Comment by barneygale** [comment]
Please pastebin the log leading up to the "server has crashed" message.

**Comment by mjgerace** [comment]
That is exactly it. The actual log is exactly the same, without the mark2 #Server Crashed! part. No errors, no problems at all with server. I disabled it and I am not crashing at all.

**Comment by barneygale** [comment]
mark2 checks whether a server has crashed by sending a newline to its console input, and checking for "unknown command" on console. If it doesn't see it 10 times in a row, it will restart. You're saying that in your mark2 log, you didn't see something like "server might have crashed, restarting in ... minutes" before mark2 restarts the server?

**Comment by mjgerace** [comment]
No, it's not my scripts file for mark2 either. I think that because I use an unsupported command in preparation for bukkit's 'playsound' command in my code, that it may be falsely picking up that message.

**Comment by trentech** [comment]
I have this problem this is what the log shows:

2013-08-16 13:08:27 | Online [0]: 
2013-08-16 13:09:27 # server might have crashed -- auto restart in 9 minutes
2013-08-16 13:09:27 | Online [0]: 
2013-08-16 13:10:27 # server might have crashed -- auto restart in 8 minutes
2013-08-16 13:10:27 | Online [0]: 
2013-08-16 13:11:27 # server might have crashed -- auto restart in 7 minutes
2013-08-16 13:11:27 | Online [0]: 
2013-08-16 13:12:27 # server might have crashed -- auto restart in 6 minutes
2013-08-16 13:12:27 | Online [0]: 
2013-08-16 13:13:27 # server might have crashed -- auto restart in 5 minutes
2013-08-16 13:13:27 | Online [0]: 
2013-08-16 13:14:27 # server might have crashed -- auto restart in 4 minutes
2013-08-16 13:14:27 | Online [0]: 
2013-08-16 13:15:27 # server might have crashed -- auto restart in 3 minutes
2013-08-16 13:15:27 | Online [0]: 
2013-08-16 13:16:27 # server might have crashed -- auto restart in 2 minutes
2013-08-16 13:16:27 | Online [0]: 

should note player count is off in case that's what your thinking. Also I'm using MCPC-Plus if that makes a difference.
